go-base/Dockerfile

```diff
@@ -2,17 +2,36 @@ FROM ubuntu:16.04
 
 RUN apt-get update 
 RUN apt-get install -y build-essential git
+
+# rumprun can only be built with gcc-6
+# XXX: Not only that, this version has to match whatever
+# gcc was used to build rumprun in the host.
+RUN apt-get update && \
```

Comment:
Let's combine this with the previous two commands into 1 `RUN` command so it will just be 1 layer. It'll help us remove the redundant `apt-get update too`

Reply:
fixed